ajax/libs/jQuery-linkify/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "jQuery-linkify",
   "description": "Intelligent link recognition, made easy",
-  "filename": "jquery.linkify.min.js",
+  "filename": "linkify.js",
```

Comment:
Why don't you directly correct it? Why did you change it into `"linkify.min.js"` in 2c45b1a3c1bf967303030e53c8e3608e5ac9269f?